pyobvector/schema/gis_func.py

```diff
@@ -0,0 +1,94 @@
+"""gis_func: An extended system function in GIS."""
+
+import logging
+from .geo_srid_point import POINT
+from sqlalchemy.ext.compiler import compiles
+from sqlalchemy.sql.functions import FunctionElement
+from sqlalchemy import BINARY, Float, Boolean, Text
+
+logger = logging.getLogger(__name__)
+
+class ST_GeomFromText(FunctionElement):
+    """ST_GeomFromText: parse text to geometry object.
+    
+    Attributes:
+    type : result type
+    """
+    type = BINARY()
+
+    def __init__(self, *args):
+        super().__init__()
+        self.args = args
+
+@compiles(ST_GeomFromText)
+def compile_ST_GeomFromText(element, compiler, **kwargs): # pylint: disable=unused-argument
+    args = []
+    for idx, arg in enumerate(element.args):
+        if idx == 0:
+            args.append(f"'{POINT.to_db(arg)}'")
```

Comment:
What if the `arg` here is not of type `Tuple[float, float]`?

Reply:
instance check added